assets/package/README.md

```diff
@@ -20,7 +20,7 @@ Install package with NPM and add it to your dependencies:
 </tr>
 <tr>
 <td>Version</td>
-<td><a href="https://github.com/mozilla/protocol/blob/master/CHANGELOG.md">19.3.0</a></td>
+<td><a href="https://github.com/mozilla/protocol/blob/master/CHANGELOG.md">20.0.0</a></td>
```

Comment:
```suggestion
<td><a href="https://github.com/mozilla/protocol/blob/main/CHANGELOG.md">20.0.0</a></td>
```

(and also on L33, that can be just shortened to `https://github.com/mozilla/protocol#readme` if you fancy)